Dockerfile

```diff
@@ -0,0 +1,12 @@
+ARG DEV_ENV_IMAGE
```

Comment:
Yes.  It needs to be available for the FROM statement just below it.

Reply:
Sorry, I meant - does the `Dockerfile` need to be in the top level directory of the repository?

Normally I try and put all the Travis CI stuff for travis in a `.travis` directory. GitHub also now lets you put most GitHub related stuff in a `.github` directory too.

Reply:
Looks like the Dockerfile can be anywhere (including outside the source repo) but everyone expects it to be at the context root (the default location). I'm not opposed to moving Dockerfile and cloudbuild.yaml to a subdirectory.  It will be less obvious to newcomers though.

Reply:
The dockerfile + cloudbuild.yaml are just for CI right?

Reply:
I wrote them to be general enough that anyone can use them to generate the database from scratch.  They do not assume anything about our CI environment.